src/web/editor.template.html

```diff
@@ -104,7 +104,6 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
 <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
 <script src="/js/loader.js"></script>
-<script src="/js/url.js"></script>
```

Comment:
So the plan would be to give all/as may as possible of these these modules similar treatment to url, right?